src/common_swagger_api/schema/tools.clj

```diff
@@ -174,6 +281,31 @@
    :date_updated           (describe Long "The timestamp of the last Tool Request status update")
    :updated_by             (describe String "The username of the user that last updated the Tool Request status")})
 
+(defschema ToolRequestListing
+  {:tool_requests (describe [ToolRequestSummary]
+                            "A listing of high level details about tool requests that have been submitted")})
+
+(defschema ToolRequestListingParams
+  (merge PagingParams
+         {(optional-key :status)
+          (describe String
+                    "The name of a status code to include in the results. The name of the status code is case
+                     sensitive. If the status code isn't already defined, it will be added to the database")}))
+
+(defschema ToolRequestStatusCodeListingParams
+  {(optional-key :filter)
+   (describe String
+             "If this parameter is set then only the status codes that contain the string passed in this
+              query parameter will be listed. This is a case-insensitive search")})
+
+(defschema ToolRequestStatusCode
+  {:id          ToolRequestStatusCodeId
```

Comment:
These `ToolRequestStatusCode*` schemas were named just `StatusCode*` in the apps service, but I thought that could get ambiguous in this lib, especially as more schemas are imported from other services.